packages/cms-cli/commands/auth.js

```diff
@@ -84,6 +83,7 @@ async function authAction(type, options) {
   }
 
   trackCommandUsage(COMMAND_NAME);
```

Comment:
I think this needs to be changed so it passes `auth` instead of `auth <type>`

Reply:
Woops, good catch